src/taxes/income-tax.ts

```diff
@@ -595,26 +622,19 @@ export function getTotalTaxAmount(
     yearsToInflate = 0,
 ): number {
     const provTax = getProvincialTaxAmount(provincialCode, grossIncome, inflationRate, yearsToInflate);
-    const provSurtax = getProvincialSurtaxAmount(provincialCode, provTax, inflationRate, yearsToInflate);
     const fedTax = getFederalTaxAmount(grossIncome, inflationRate, yearsToInflate);
-
-    return provTax + provSurtax + fedTax;
+    const abatement = getProvincialAbatement(provincialCode, fedTax);
+    return Math.max(provTax, 0) + Math.max(fedTax - abatement, 0);
 }
 
-export function calculateEffectiveTaxRate(income: number, province: ProvinceCode): number {
+export function calculateEffectiveTaxRate(
+    income: number,
+    province: ProvinceCode,
+    inflationRate = 0,
+    yearsToInflate = 0,
```

Comment:
Je crois qu'on devrait changer l'ordre des paramètres pour que ça ressemble plus au autres fonctions
```suggestion
    province: ProvinceCode,
    income: number,
    inflationRate = 0,
    yearsToInflate = 0,
```

Reply:
Par contre, puisque la fonction est utilisé dans express, idéalement faudrait que lorsqu'on bump la version dans express, on fasse attention de bien corriger cela.

Reply:
Ouais j'avais remarqué et je ne l'ai pas changé pour les effets "incontrôlé" d'inverser les paramêtres. On peut se gâter si t'es confiant.

Reply:
Je suis confiant, les gens qui font un update on juste à checker leurs affaires